chart/templates/deployment.yaml

```diff
@@ -22,6 +22,14 @@ spec:
         image: {{ .Values.scannerImage}}:{{ .Values.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
         env:
+        - name: GOMEMLIMIT
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.memory
```

Comment:
similar for this. I think this has to be an `int64` in Go, and I don't know if this is required in k8s. Also, the `Suggested Uses` here https://tip.golang.org/doc/gc-guide#Memory_limit mentions we should leave 5-10% of wiggle room